renew/utils.go

```diff
@@ -0,0 +1,129 @@
+package renew
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"cloud.google.com/go/firestore"
+	"github.com/wopta/goworkspace/lib"
+	"github.com/wopta/goworkspace/models"
+	"google.golang.org/api/iterator"
+)
+
+// TODO: remove me
+const (
+	policyRenewedTestCollection      string = "policyRenewedTest"
+	transactionRenewedTestCollection string = "transactionRenewedTest"
+)
+
+func createSaveBatch(policy models.Policy, transactions []models.Transaction) map[string]map[string]interface{} {
+	policy.BigQueryParse()
+	batch := map[string]map[string]interface{}{
+		policyRenewedTestCollection: { // TODO: change to lib.PolicyCollection
+			policy.Uid: policy,
+		},
+		transactionRenewedTestCollection: {}, // TODO: change to lib.TransactionCollection
+	}
+
+	for idx, tr := range transactions {
+		tr.UpdateDate = time.Now().UTC()
+		tr.BigQueryParse()
+		batch[transactionRenewedTestCollection][tr.Uid] = tr // TODO: change to lib.TransactionCollection
+		transactions[idx] = tr
+	}
+
+	return batch
+}
```

Comment:
SUGGESTION: maybe we could think to a `dryRun` variable to determine in which is the target collection